test.js

```diff
@@ -30,12 +51,13 @@ test('should be able to connect and perform a query', t => {
   const fastify = Fastify()
 
   fastify.register(fastifyPostgres, {
+    name: 'test',
     connectionString: 'postgres://postgres@localhost/postgres'
   })
 
   fastify.ready(err => {
     t.error(err)
-    fastify.pg.connect(onConnect)
+    fastify.pg.test.connect(onConnect)
```

Comment:
I would not change the tests, and keep using the non-prefixed version. But add a new test to check on the prefixed version.

Reply:
got it